masterlist.yaml

```
@@ -618,6 +618,12 @@ common:
       - '22.6'
       - '[xOBSE](https://github.com/llde/xOBSE/)'
     condition: 'version("../obse_1_2_416.dll", "0.0.22.6", <)'
+  - &versionXofYorGreaterRequired_xOBSEv22_8
+    <<: *versionXofYorGreaterRequired
+    subs:
+      - '22.8'
+      - '[xOBSE](https://github.com/llde/xOBSE/)'
+    condition: 'version("../obse_1_2_416.dll", "0.0.22.8", <)'
```

Looks like the version format changed at some point:

```suggestion
    condition: 'version("../obse_1_2_416.dll", "0.22.8.0", <)'
```